### DISCUSSION TITLE
[A/L] (2021/22) Funzioni - Esercizio 24

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#264] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152317094-52f9a744-e97c-4ecd-a73f-a0c5d6e81c71.png)

**Comment by LuginiAndrea** [comment]
La prima è falsa in quanto, poichè # $B$ < # $A$, $g$ non può essere suriettiva (basti provare ad associare ad ogni numero una lettera diversa in quanto $g$ non sarebbe una funzione in caso contrario)

La seconda è vera
$$g: a \rightarrow 1, b \rightarrow 2, c \rightarrow 3, d \rightarrow 4$$
$$f: 1 \rightarrow a, 2 \rightarrow b, 3 \rightarrow c, 4 \rightarrow d, 5 \rightarrow d$$
Possiamo vedere che $f \circ g$ è l'identità su $B$

La terza pure è palesemente vera: infatti, presa qualsiasi definizione di $f$, ogni immagine di $f$ viene mandata a 1 dalla funzione $g$
